roles/cinder-common/defaults/main.yml

```diff
@@ -51,3 +54,9 @@ cinder:
     debug: False
     verbose: True
   cafile: "{{ ssl.cafile|default('/etc/ssl/certs/ca-certificates.crt') }}"
+  ceph:
+    rbd_user: cinder
+    rbd_ceph_conf: /etc/ceph/ceph.conf
+    rbd_flatten_volume_from_snapshot: false
+    rbd_max_clone_depth: 5
+    glance_api_version: 2
```

Comment:
@j2sol I'd rather move all these into `ceph-common/defaults/main.yml`, even though they are used by the cinder role. Thoughts?